app/views/home/temp_dashboard.html.erb

```diff
@@ -0,0 +1,22 @@
+<div id="dashboard">
+  <div id="page">
+    <div class="banner">
+      <iron-icon icon="icons:error" class="alert"></iron-icon>
+ You've been assigned as an author on a new paper. <a href="#">Begin review.</a>
+    </div>
+    <div class="papers">
+      <h2 class="headline">Hi <%= current_user.name %>. You have <%= link_to(pluralize(author_papers.size, 'paper'), "#submitted") %> submitted, <%= link_to(pluralize(reviewer_papers.size, 'paper'), "#reviewing") %> you're reviewing and <%= link_to(pluralize(editor_papers.size, 'paper'), "#editing") %> you're editing.</h2>
```

Comment:
I think it would be worth adding a bit more intelligence here. I don't think we should show editing counts if they are not an editor (or have a zero count alternatively).